01 - Getting Started.ipynb

```diff
@@ -24,13 +24,13 @@
     "### `conda` / `mamba`\n",
```

Comment:
<p>We haven't mention <code>.drop</code> before the exercise, I think it's probably fine, but just FYI.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ibis-project/ibis-tutorial/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
<p>yeah, I'll leave this for now, but I can mention during the tutorial that <code>drop</code> does what you would expect it does </p>